src/main/java/com/gmail/inverseconduit/commands/sets/AdminCommands.java

```diff
@@ -0,0 +1,152 @@
+package com.gmail.inverseconduit.commands.sets;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import com.gmail.inverseconduit.BotConfig;
+import com.gmail.inverseconduit.commands.CommandHandle;
+
+public class AdminCommands {
+
+	public static CommandHandle addAdminCommand(BotConfig bc) {
+		return new CommandHandle.Builder("addAdmin", message -> {
+
+			if (isElevatedUser(message.getUserId(), bc)) {

```

Comment:
this if-condition could be inverted into a so-called "guard clause" to reduce the level of indentation in the following code:

    if (!isElevatedUser(message.getUserId(), bc)) {
        return "I'm afraid I cannot let you do that";
    }